product-mini/platforms/android/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ set (ANDROID_NDK $ENV{ANDROID_NDK_HOME})
 set (ANDROID_SDK $ENV{ANDROID_SDK_HOME})
 set (ANDROID_ABI "x86")
 set (ANDROID_LD lld)
+set (ANDROID_PLATFORM 24)
```

Comment:
Seems that it isn't used, in fact the macro __ANDROID_API__  is used instead, could we remove this line?

Reply:
Some reasons make think we should "set (ANDROID_PLATFORM 24)" by default.
1.  refer to [Android NDK ANDROID PLATFORM Doc](https://developer.android.com/ndk/guides/cmake#android_platform),
seems that we should set "ANDROID_PLATFORM" to determine the value of macro __ANDROID_API__.
2. API level 24, equals Android system version 7.0, most of current android system version is larger that 7.0 .
![image](https://user-images.githubusercontent.com/10172640/112263058-a1645180-8ca9-11eb-98b7-45c8b2dfafd0.png)
In that situation, wamr libc WASI could supports most of APIs except getrandom().
(if we remove this line, instead we could tell developers that they must config "ANDROID_PLATFORM" if they want to support libc WASI  in the WAMR developer Document). 

If need, I can remove this line. How do you think? thanks.

Reply:
Got it, but had better not hardcode it to 24, could you please modify it to
```bash
if (NOT DEFINED ANDROID_PLATFORM)
    set (ANDROID_PLATFORM 24)
endif ()
```
So developer can use `cmake -DANDROID_PLATFORM=xx` to set the version number if his version number is different.

Reply:
Ok, I'll change and test it later.